flink-runtime/src/test/java/org/apache/flink/runtime/filecache/FileCacheReadsFromBlobTest.java

```diff
@@ -33,16 +38,16 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.util.concurrent.Future;
 
-import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
 
 /**
- * Test delete process of {@link FileCache}. The local cache file should not be deleted why another task comes in 5 seconds.
+ * Tests that {@link FileCache} can read files from {@link BlobServer}.
  */
-public class FileCacheDeleteValidationTest {
+public class FileCacheReadsFromBlobTest {
```

Comment:
am i missing something, or isn't this now a duplicate of `FileCacheDirectoryTest#testDirectoryDownloadedFromBlob`

Reply:
It is different in the way that this test uploads a single file. Therefore it follows a bit different path. The file is not zipped and is directly shipped from BlobCache. In case of directories there is unzipped version which lifecycle is managed by `FileCache`.

If you think though the test `FileCacheDirectoryTest#testDirectoryDownloadedFromBlob` is enough, I will remove it.

Reply:
it does, it is `FileCache` responsibility to unzip.

Reply:
ah yes, i was misreading the test...
´